src/core_ocean/shared/mpas_ocn_tracer_hmix_redi.F

```diff
@@ -332,7 +335,7 @@ subroutine ocn_tracer_hmix_Redi_tend(meshPool, scratchPool, layerThickness, laye
                ! Add tendency for Term 3: d/dz ( h S grad phi) = ( S grad phi) fluxes
                ! 2.0 in next line is because a dot product on a C-grid
                ! requires a factor of 1/2 to average to the cell center.
-               flux_term3 = RediKappa(iCell)*2.0_RKIND* &
+               flux_term3 = rediKappaCell(iCell)*2.0_RKIND* &
                             (kappaRediSfcTaper(k, iCell)*kappaRediCell(k, iCell)*fluxRediZTop(iTr, k)  &
```

Comment:
`rediKappaCell` needs a `k` index. But more generally, term 3:
![image](https://user-images.githubusercontent.com/4036816/86825442-964e9000-c04c-11ea-9c2f-369a6b45cb13.png)
should only have one factor of kappa, and this line has kappa^2.

Reply:
actually it doesn't.  there are two different arrays, one is kappaRediCell (one line down) this is rediKappaCell, which is the constant value interpolated from edges to centers.  I agree it's a poor naming convention.  Any suggestions how I could rename?